rllib/policy/policy.py

```diff
@@ -1269,19 +1280,37 @@ def _initialize_loss_from_dummy_batch(
         for key, value in extra_outs.items():
             self._dummy_batch[key] = value
             if key not in self.view_requirements:
-                self.view_requirements[key] = ViewRequirement(
-                    space=gym.spaces.Box(
-                        -1.0, 1.0, shape=value.shape[1:], dtype=value.dtype
-                    ),
-                    used_for_compute_actions=False,
-                )
+                if isinstance(value, np.ndarray):
```

Comment:
See my comment below on generalizing the `get_gym_space_from_struct_of_tensors` utility to also accept a single tensor.